snappy_pipeline/workflows/panel_of_normals/__init__.py

```diff
@@ -118,54 +116,82 @@
 class PanelOfNormalsStepPart(BaseStepPart):
     """Base class for panel of normals step parts
 
-    Variant calling is performed on matched cancer bio sample pairs.  That is, the primary NGS
-    library for the primary bio sample is used for each cancer bio sample (paired with the primary
-    normal bio sample's primary NGS library).
+    Two steps: the preparation is done on each normal samples separately, and the panel creation
+    merges all the individual results in the the panel.
     """
 
+    #: Class available actions
+    actions = ("run", "prepare_panel", "create_panel")
```

Comment:
Where are methods that implement the action 'run'?

Reply:
There aren't. I am not sure whether `run` is necessary or not. If it is, should I create input, output & log functions? And if yes, are the methods supposed to trigger an exception?

Reply:
_Short answer_: I don't know if you need it.

If you don't need it, you can just remove it. But if I see an available action listed, I expect it to be implemented.

You can double check in the `Snakefile` if there is a rule associated with the action 'run'. 
Or even better, just build the DAG from the workflow and visually inspect it:
```sh
snakemake --rulegraph --snakefile /path/to/development/snappy/workflow/Snakefile --directory /path/to/workflow/working/directory  --dry-run  --debug --verbose  | dot -Tsvg > dag_WORKFLOW.svg
```